data/json/monstergroups/military.json

```
@@ -123,7 +127,8 @@
       { "monster": "mon_zombie_military_pilot", "weight": 20 },
       { "monster": "mon_zombie_flamer", "weight": 1, "cost_multiplier": 30 },
       { "monster": "mon_zombie_scientist", "weight": 100 },
-      { "monster": "mon_feral_soldier", "weight": 75 }
+      { "monster": "mon_feral_soldier", "weight": 75 },
+      { "monster": "mon_feral_soldier_flamer", "weight": 45 }
```

```suggestion
      { "monster": "mon_feral_soldier_flamer", "weight": 1 }
```
Matching zombie relationships